internal/dataplane/parser/translate_httproute.go

```diff
@@ -29,6 +31,46 @@ func (p *Parser) ingressRulesFromHTTPRoutes() ingressRules {
 		return result
 	}
 
+	if p.featureFlags.ExpressionRoutes {
+		splittedHTTPRoutes := []*gatewayv1beta1.HTTPRoute{}
```

Comment:
`splittedHTTPRoutes` -> `splitHTTPRoutes`

Reply:
changed all `splitted` to `split`.